### ISSUE TITLE
Editor's Create Table popup is not accessible for screen readers

### ISSUE THREAD

**Issue by martintabakov** [telerik/kendo-ui-core#7736]
### Bug report

The Create Table popup is not accessible for screen readers when you select the table size using the keyboard.

**Workaround**: Assign aria-live="polite" attribute to the k-status element:

`$('[title="Create a table"]').click(function(){
  	$(".k-status").attr("aria-live", "polite")
  })`

### Reproduction of the problem

1. Open this Dojo example with a screen reader open (e.g. NVDA) - https://dojo.telerik.com/enAFIvab
2. Click on the Create Table button
3. Change the table size using the arrow keys.

### Current behavior

The cell selection is not announced when using the arrow keys

### Expected/desired behavior

The table size should be announced after changing the cell selection

### Environment

* **Kendo UI version:** 2024.1.130 
* **Browser:** [all]
